includes/payment-methods/class-upe-payment-method.php

```diff
@@ -281,7 +301,12 @@ public function get_payment_token_for_user( $user, $payment_method_id ) {
 	 * @param string $account_country The country of the account.
 	 * @return string
 	 */
-	abstract public function get_testing_instructions( string $account_country );
+	public function get_testing_instructions( string $account_country ) {
+		if ( null !== $this->definition ) {
+			return $this->definition::get_testing_instructions();
```

Comment:
It looks like we might want to add the `$account_country` argument to the method in `PaymentMethodDefinitionInterface`?

Reply:
Yeah, that should definitely be there. Added in cb001dd